site/src/templates/organizations.js

```diff
@@ -1,30 +1,53 @@
-import React from "react"
+import React, { useEffect } from "react"
 import { graphql } from "gatsby"
 import { uniqBy } from "lodash"
+import { useLazyQuery } from "@apollo/react-hooks"
+import gql from "graphql-tag"
 
 import { transformCategories, transformOrganizations } from "../utils/airtable"
+import { useAuth0 } from "../components/Auth0Provider"
 
 import Layout from "../components/layout"
 import OrganizationCard from "../components/OrganizationCard"
 import IndexHeader from "../components/IndexHeader"
 import { useOrganizationFilterState } from "../components/OrganizationFilter"
 import SEO from "../components/seo"
 import CategoryList from "../components/CategoryList"
+import { GetFavorites, indexFavoritesData } from "../components/FavoriteButton"
 
 function OrganizationsTemplate({
   data,
   pageContext: { categoryId, categoryName, categoryCounts },
 }) {
   const [filter, setFilter, applyFilter] = useOrganizationFilterState()
 
+  const { loading: authLoading, user } = useAuth0()
+
+  const [getFavorites, { data: favoritesData, error: favoritesError }] = useLazyQuery(GetFavorites, {
+    variables: {
+      loggedIn: !!user,
+      userId: user?.sub,
+    },
+  })
+
+  if (favoritesError) console.error(favoritesError)
+
+  useEffect(() => {
+    if (!authLoading) getFavorites()
+  }, [authLoading])
+
   // We need to combine organizations from the query for sub-categories
   // and top-categories which might include duplicate orgs.
   const orgs = uniqBy(
     [...data.subOrganizations?.nodes, ...data.topOrganizations?.nodes],
-    org => org.data.Name
+    org => org.recordId
   )
 
-  const allOrganizations = transformOrganizations(orgs)
+  const favorites = indexFavoritesData(favoritesData)
+  const allOrganizations = transformOrganizations(orgs, (raw, org) => ({
```

Comment:
This let's us easily pass the `favorites` data through to each `OrganizationCard` and `FavoriteButton`